hera_cal/lst_stack/calibration.py

```diff
@@ -335,6 +341,51 @@ def _lstbin_phase_calibration(
     return calibration_parameters, phase_gains
 
 
+def _lstbin_cross_pol_phase_calibration(
+    stack: LSTStack,
+    model: np.ndarray,
+):
+    """
+    This function performs cross-polarization phase calibration on LSTStack object by comparing each day to an input
+    reference model.
```

Comment:
I'd at least mention what the output is and what it's units are

Reply:
Fixed this